src/Components/FirmwareSelector/index.jsx

```diff
@@ -298,6 +308,7 @@ function FirmwareSelector({
 
           <div className="spacer-box">
             <LabeledSelect
+              disabled={layoutSelectionDisabled}
```

Comment:
Please do not disable this. In case of AM32 you can't chose anything else here anyway.